src/test/steps/IdamUserManager.ts

```diff
@@ -89,6 +94,17 @@ export class IdamUserManager {
     this.users = new Set([firstUser]);
   }
 
+  static async getAccessTokenFromIdam(username: string, password: string): Promise<AxiosResponse<OidcResponse>> {
+    const id: string = sysConfig.get('services.idam.clientID');
+    const secret = sysConfig.get('services.idam.clientSecret');
+    const tokenUrl: string = sysConfig.get('services.idam.tokenURL');
+
+    const headers = { Accept: 'application/json', 'Content-Type': 'application/x-www-form-urlencoded' };
+    const data = `grant_type=password&username=${username}&password=${password}&client_id=${id}&client_secret=${secret}&scope=openid%20profile%20roles%20openid%20roles%20profile`;
```

Comment:
I know this is how it was setup before, but can we use URLSearchParams instead of appending everything to the url 
it should make it much more readable (unless it doesn't actually work)